reference-architecture/aws-ansible/playbooks/roles/cloudformation-infra/files/brownfield-byo-bastion.json

```diff
@@ -708,6 +708,9 @@
             },
             { "Key": "openshift-role",
               "Value": "master"
+            },
+            { "Key": "stack-name",
+              "Value": {"Ref": "AWS::StackName"}
```

Comment:
That said, I think they already get assigned a dedicated tag associated with the stack: `aws:cloudformation:stack-name`